src/devicewidget.h

```diff
@@ -25,6 +25,7 @@
 
 #include "minimalstreamwidget.h"
 #include "ui_devicewidget.h"
+#include <QAction>
```

Comment:
Please, use just forward declarations here... (and add the include into .cpp, if needed)